src/main/kotlin/io/openfuture/state/Application.kt

```diff
@@ -9,4 +9,3 @@ class Application
 fun main(args: Array<String>) {
     runApplication<Application>(*args)
 }
-
```

Comment:
Files should end with an empty line always https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729725

Reply:
Check other files as well

Reply:
This file had two empty lines, so i removed one